src/SFA.DAS.AssessorService.Application/Handlers/Staff/GetLearnerDetailHandler.cs

```diff
@@ -78,15 +77,18 @@ public class GetLearnerDetailHandler : IRequestHandler<GetLearnerDetailRequest,
                 epao = await _organisationRepository.Get(certificate.OrganisationId) ?? new Organisation();
             }
 
+            var standard = await _standardRepository.GetStandardVersionByLarsCode(request.StdCode, certificateData.Version);
```

Comment:
May want to add a null conditional onto certificateDate?.Version, as if the certificate is null you won't get the cert data.